python/ray/data/_internal/execution/interfaces/physical_operator.py

```diff
@@ -183,6 +183,7 @@ def __init__(
         self._target_max_block_size = target_max_block_size
         self._started = False
         self._in_task_submission_backpressure = False
+        self._max_task_output_bytes_to_read = None
```

Comment:
```suggestion
        self._latest_max_task_output_bytes_to_read = None
```
Or maybe just `_in_task_output_backpressure` for consistency with the above flag.

Reply:
Ah I guess it probably makes more sense to just store the flag as a boolean rather than store the actually number of output bytes to read. Will update and go with `_in_task_output_backpressure` for consistency.